anchore_engine/services/policy_engine/api/controllers/synchronous_operations.py

```diff
@@ -501,23 +501,15 @@ def save(self, result):
     def _inputs_changed(self, cache_timestamp):
         # A feed sync has occurred since the eval was done or the image has been updated/reloaded, so inputs can have changed. Must be stale
         db = get_session()
-        # TODO: zhill - test more
-        feed_group_updated_list = [
-            group.last_sync
-            if group.last_sync is not None
-            else datetime.datetime.utcfromtimestamp(0)
-            for feed in get_all_feeds(db)
-            for group in feed.groups
-        ]
-        feed_synced = (
-            max(feed_group_updated_list) > cache_timestamp
-            if feed_group_updated_list
-            else False
-        )
 
         image_updated = self.image.last_modified > cache_timestamp
 
-        return feed_synced or image_updated
+        return (
+            image_updated
```

Comment:
it looks like the order of the returned values was changed, was that on purpose?

Reply:
Yep, both the checks are relatively light weight. That said, checking if the image was updated may be slightly faster relative to the other check which is now delegated to the provider. Re-ordered the checks in the perf order so that the logic can bail out quickly if the 1st check fails